docs/configure_admin_workstation_post_install.rst

```diff
@@ -63,3 +63,8 @@ of the authenticated hidden services, restart Tails and make
 sure to enable persistence.
 
 .. _Authenticated Tor Hidden Services: https://www.torproject.org/docs/tor-manual.html.en#HiddenServiceAuthorizeClient
+
+Back Up The Workstations
```

Comment:
Nit: The -> the (title case per https://docs.securedrop.org/en/release-0.12.1/development/documentation_guidelines.html although we're admittedly not super-consistent about this).